File: basics/caesar.py
def caesar(text, shift):
    alphabets = "abcdefghijklmnopqrstuvwxyz"
    shifted_text = alphabets[shift:] + alphabets[:shift]
    table = str.maketrans(alphabets, shifted_text)
    return text.translate(table)


encrypted_text_2 = caesar("freecodecamp", 5)
encrypted_text = caesar("freeCodeCamp", 5)
print(encrypted_text)
print(encrypted_text_2)

File: 01-basics/caesar.js
function caesar(text, shift) {
    let alphabets = "abcdefghijklmnopqrstuvwxyz!@#$"
    for (let i = 0; i < text.length; i++) {
        console.log(alphabets[(alphabets.indexOf(text[i]) + shift)])
    }
}

caesar("saurabh", 5)